background_script.js

```diff
@@ -166,5 +166,18 @@
       );
     });
   };
+
+  chrome.browserAction.onClicked.addListener(async (info, tab) => {
+    console.log("Clicked");
+    if (!SUPPORTS_TEXT_FRAGMENTS) {
+      await polyfillTextFragments();
+    }
+    await injectContentScripts([
+      'prepare.js',
+      'fragment-generation-utils.js',
+      'content_script.js',
+    ]);
+    startProcessing(tab);
+  });
```

Comment:
Why is this needed?

Reply:
This is added to enable "clicking the icon to copy". Basically a copy of the same operations for corresponding click from the context menu.

Reply:
I see. In this case, can you please factor out the code in https://github.com/GoogleChromeLabs/link-to-text-fragment/blob/27728509b02305e91d0483fa8774c03796bafe6f/background_script.js#L121-L131 into a function and call it from both locations, `browser.contextMenus.onClicked` and `browser.browserAction.onClicked` (note that I have re-name-spaced `chrome` to `browser`.

Reply:
Nice refactoring suggestion! Will do this.

Reply:
Done in 8010cf8bb91af6ca05c3a98762981120f4682a20